bower.json

```diff
@@ -21,18 +21,37 @@
   "homepage": "https://github.com/PolymerElements/gold-cc-cvc-input",
   "ignore": [],
   "dependencies": {
-    "iron-icon": "PolymerElements/iron-icon#^1.0.0",
-    "iron-flex-layout": "PolymerElements/iron-flex-layout#^1.0.0",
-    "iron-form-element-behavior": "PolymerElements/iron-form-element-behavior#^1.0.0",
-    "paper-input": "PolymerElements/paper-input#^1.0.0",
-    "paper-styles": "PolymerElements/paper-styles#^1.0.0",
-    "polymer": "Polymer/polymer#^1.1.0"
+    "iron-icon": "PolymerElements/iron-icon#2.0-preview",
```

Comment:
can you please update these to `1 - 2`, `1.9 - 2` and add the resolutions etc.

[paper-input](https://github.com/PolymerElements/paper-input/blob/master/bower.json) has a good example